frontend/test/fixtures/Copy.json

```diff
@@ -0,0 +1,23 @@
+{
+  "swrFallback": {
+    "$inf$#query:\"\",offset:0,limit:18,": [
```

Comment:
Doesn't seem like a good idea to have the cache key in a fixture. Does the test actually need to check the `swrFallback`? Does a user care what's in the `swrFallback`?

Reply:
Resolved in 2c0fcbb9.